### DISCUSSION TITLE
Install autopilot on the pilot, numpy incompatibility

### DISCUSSION THREAD

**Discussion by neurodavidus** [wehr-lab/autopilot#77] [Q&A] [answered]
Hello 
During the install of autopilot I got this 
![image](https://user-images.githubusercontent.com/4154936/107778523-74d12780-6d44-11eb-89ef-8a3253890633.png)
Not sure how important this numpy requirement will be problematic....
Thanks,  David

**Comment by sneakers-the-rat** [comment, marked as answer]
Looks like you're using the system python interpreter, and it looks like you have an old version of numpy installed (probably through apt, like `apt install python3-numpy`, since that's the latest version on the raspberry pi apt repos). installing fresh in a virtualenv should fix, or you can otherwise update system numpy by using `pip3 install --upgrade numpy` :) I'll add that to the requirements files now

**Reply by neurodavidus** [reply]
Just realized I forgot to say that you were (as always) correct and that this is solved. thanks
